### PR TITLE
Api tools

### PR DESCRIPTION
**Pull request by ttheune**
Four sample scripts for users to use the API to do some basic IT tasks.

### CODE REVIEW

**Comment by rjo100** [comment]
I like it

File: api/foxpass_deactivate_user.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import json
+import requests
+
+URL = 'https://api.foxpass.com/v1/'
+ENDPOINT = 'users/'
+DATA = {'is_active': False}
+
+def main():
+    parser = argparse.ArgumentParser(description='List groups in Foxpass')
+    parser.add_argument('--key', required=True, help='Foxpass API Key')
```
Comment (aren) [inline api/foxpass_deactivate_user.py]: I think all (most?) of our other scripts call this --api-key.

**Comment by aren** [comment]
minor comment, otherwise awesome!

**Comment by ttheune** [comment]
Valid point, fixed.

**Pull request merged**